### ISSUE TITLE
Preview page

### ISSUE THREAD

**Issue by hrna** [hrna/cvtool#4]
Should there be a preview page before creating CV ?
Here user could choose which format he would like to have the CV (PDF, DOCX, JSON..etc..)

**Comment by kgorecki** [comment]
I think that preview would be really cool, however I don't know how reliable will it be.

**Comment by hrna** [comment]
Going to try this approach and trash it if it wont work out.

**Comment by kgorecki** [comment]
Makes sense. Actually even basic preview of content would be good. Sometimes you can't see typos etc. if there are too many things on the screen (frames, fields, buttons etc.) so clean place for verification would be really handy.
